Pipfile

```diff
@@ -14,6 +14,8 @@ flake8-docstrings = "*"
 pre-commit = "*"
 variant-normalization = {editable = true, path = "."}
 pyyaml = "*"
+jupyter = "*"
+ipykernel = "*"
```

Comment:
could these be in the dev requirements rather than the main project requirements?

Reply:
Why do I always forget to use `--dev`???

Reply:
Oh wait it looks like it's already in dev?